modules/ROOT/pages/7.5-release-notes.adoc

```diff
@@ -91,7 +91,24 @@ For information on the **Markdown** plugin, see xref:markdown.adoc[Markdown].
 
 The {productname} {release-version} release includes an accompanying release of the **Comments** premium plugin.
 
-**Comments** Premium plugin includes the following fixes.
+**Comments** Premium plugin includes the following additions and fixes.
+
+==== Comments with Mentions
+// EPIC-111
+
+The **Comments** plugin now supports mentions in comments.
+
+Previously, the **Comments** plugin allowed users to add comments to content in the editor. However, users could not mention other users in the comments sidebar, limiting the ability to notify specific users or direct comments to them.
+
+In {productname} {release-version}, the **Comments** plugin has been enhanced to support xref:mentions.adoc[Mentions] in comments. Users can now mention other users in comments by typing the `+@+` symbol followed by the user's name. This feature enables users to notify specific individuals, direct comments to them, or engage in conversations with other users directly within the editor.
+
+Additionally, the **Comments** plugin now includes a new `mentionedUids` property in the EventLog API when the Mentions plugin is enabled. This property provides an array of UIDs mentioned in the comment, allowing integrators to retrieve which users have been mentioned in comments.
+
+The **Comments** plugin now offers the `xref:comments-with-mentions.adoc#tinycomments_mentions_enabled[tinycomments_mentions_enabled]` option to enable or disable the incorporation of the Mentions plugin into the **Comments** plugin, provided both plugins are included in the configuration.
+
+liveDemo::comments-embedded-with-mentions[]
```

Comment:
Do we normally include live demos in release notes?

Reply:
After a discussion with @FarzadHayat, I'm comfortable including this for now. However, if you think we should add a note like "To test out comments with mentions, see [link]," instead, im also good with this.

Reply:
Up to you guys, just wanted to flag it.